src/routes/jira/connect/enterprise/jira-connect-enterprise-post.ts

```diff
@@ -27,6 +27,7 @@ export const gheServerUrlErrors: GheServerUrlErrors = {
 		message: "Invalid URL"
 	},
 	ENOTFOUND: GHE_ERROR_UNREACHABLE,
+	DEPTH_ZERO_SELF_SIGNED_CERT: GHE_ERROR_UNREACHABLE,
```

Comment:
Will it just say that the server is unreachable? Shouldn't we be more specific? (Or the message would contain the code itself that might help the customer?)

Reply:
This isn't what we render in the UI. We're rendering a more specific message with a link to documentation. The backend isn't responsible for what we surface users in the client. Only for sending the error.

Reply:
Ah right, sorry, I didn't check where the change was made 🤦